tests/unittests/EntryPoint.cpp

```diff
@@ -1,6 +1,6 @@
 //******************************************************************************
 //
-// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Just curious, why remove the year instead of updating it to 2016? Is this a new policy going forward?